### PR TITLE
Fb/create db models

### PR DESCRIPTION
**Pull request by BrandonMagana**
# Create database Models

This is the list of models created:
- Donation
-  Event
-  Inventory
-  Address
-  Contributor (refactor)

### CODE REVIEW

File: server/food_bank_app/donations/views.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from rest_framework.permissions import IsAuthenticated
+from contributors.exceptions import ContributorExistsException, ContributorNotSetException
+from contributors.selectors import select_contributor
+#from .serializers import ContributorSerializer
+#from .services import create_contributor
+
+class ContributorViewSet(ViewSet):
+    permission_classes = IsAuthenticated,
+    """
+    def retrieve(self, request):
```
Comment (antonieto) [inline server/food_bank_app/donations/views.py]: Uncomment?

File: server/food_bank_app/donations/views.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from rest_framework.response import Response
+
+# Create your views here.
+from rest_framework.permissions import IsAuthenticated
+from contributors.exceptions import ContributorExistsException, ContributorNotSetException
+from contributors.selectors import select_contributor
+#from .serializers import ContributorSerializer
+#from .services import create_contributor
+
+class ContributorViewSet(ViewSet):
```
Comment (CarlosCRG19) [inline server/food_bank_app/donations/views.py:11]:
```suggestion
class DonationViewSet(ViewSet):
```

Comment (BrandonMagana) [reply server/food_bank_app/donations/views.py]: my bad dummie file

File: server/food_bank_app/donations/models.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from events.models import Event
+from contributors.models import Contributor
+
+import uuid
+
+class Donation(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    contributor = models.OneToOneField(Contributor, on_delete=models.SET_NULL)
```
Comment (antonieto) [inline server/food_bank_app/donations/models.py]:
```suggestion
    contributor = models.ForeignKey(Contributor, on_delete=models.SET_NULL)
```

**Review by antonieto — APPROVED** [review]
lgtm

**Review by CarlosCRG19 — APPROVED** [review]

**Pull request merged**